lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -14,8 +14,11 @@
  * 0x70000000 - 0x7FFFFFFF: Reserved for implementation-specific return
  *			    code providing non-error information
  * 0x80000000 - 0x8FFFFFFF: Reserved for implementation-specific errors
+ *
+ * TEE_ERROR_DEFER_DRIVER_INIT - Device driver failed to initialize because
+ *	it depends on a device not yet initialized.
```

Comment:
What's "it" supposed to align with? It seems we have two choices, either "TEE_ERROR_DEFER_DRIVER_INIT" or "Device" at the row above.

Reply:
Ok i'll rephrase more explicitly:
```
 * TEE_ERROR_DEFER_DRIVER_INIT - Device driver failed to initialize because
 *	the driver depends on a device not yet initialized.